docs/pages/index.mdx

```diff
@@ -46,16 +53,57 @@ import arrowRightSvg from "@site/src/components/Icon/teleport-svg/arrow-right.sv
   ]}
 />
 
-Teleport is the easiest, most secure way to access and protect all your infrastructure.
-
-The Teleport Infrastructure Identity Platform implements trusted computing at
-scale, with unified cryptographic identities for humans, machines and workloads,
-endpoints, infrastructure assets, and AI agents. 
-
-## Get started
-
-You can quickly see how Teleport works by following our [Get
-Started](./get-started/get-started.mdx) guide to enroll your first resource with Teleport.
+<GetStarted
+  youtubeVideoId="BJWbSqiDLeU"
+  steps={[
+    {
+      title: "Step 1: Deploy a Teleport cluster",
+      description: "Select an edition and deploy a Teleport cluster. ",
+      href: "./get-started/",
+    },
+    {
+      title: "Step 2: Connect your infrastructure",
+      description: "Connect and protect your infrastructure with your Teleport cluster.",
+      href: "./get-started/connect/",
+    },
+    {
+      title: "Step 3: Manage access",
+      description: "Configure role-based access controls for your team.",
+      href: "./get-started/access/",
+    },
+    {
+      title: "Step 4: Review a detailed audit log",
+      description: "Collect detailed events generated by humans and machines across your infrastructure.",
+      href: "./get-started/audit/",
+    },
+  ]}
+  links={[
+    {
+      title: "Installation guide",
+      description: "Detailed guide on how to install Teleport binaries/clients on your platform.",
+      href: "./installation/",
+      iconComponent: bookOpenSvg,
+    },
+    {
+      title: "Architecture",
+      description: "Understand how Teleport is architected and how its components work together.",
+      href: "./reference/architecture/",
+      iconComponent: flowArrowSvg,
+    },
+    {
+      title: "User guides",
+      description: "Guides for accessing infrastructure with Teleport tools and interfaces.",
+      href: "./connect-your-client/",
+      iconComponent: identificationBadgeSvg,
+    },
+    {
+      title: "References",
+      description: "A complete references for configuring core components across environments.",
```

Comment:
```suggestion
      description: "Comprehensive guides for configuring Teleport components across environments.",
```
- Avoids repeating the title in the description
- I'm not sure what "core" means here, so I've suggested replacing it with "Teleport"